tensorboard/plugins/scalar/scalars_demo_eager.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Write TensorBoard scalar summary data.
+
+This short example illustrates how to use the scalar writing in eager
+mode TensorFlow 2.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+from absl import app
+from absl import flags
+from absl import logging
+
+from six.moves import xrange  # pylint: disable=redefined-builtin
+import tensorflow as tf
+from tensorboard.plugins.scalar import summary
+import random
```

Comment:
Can we adhere to Google's style guide w.r.t. ordering/grouping of imports?
e.g. `import random` should move up next to `import os` since it's a stdlib package

[1] https://google.github.io/styleguide/pyguide.html#313-imports-formatting

Reply:
Done.